frontend/src/pages/dashboard/components/tables/trucks-table/trucks-table.tsx

```diff
@@ -94,15 +97,19 @@ const TruckTable: React.FC = () => {
           pageSize={pageSize}
           setSorting={setSorting}
           sorting={sorting}
-          isLoading={dataStatus === DataStatus.PENDING}
+          isLoading={dataStatus === DataStatus.IDLE}
```

Comment:
Why is there an IDLE here? In this case the loader will be only once when the component is initialized.

Reply:
Yes, you're right, this is a mock. I don't think isLoading is needed here at all because the table reloads with each form submission.